distributed/tests/test_core.py

```diff
@@ -771,8 +771,7 @@ async def f():
         await server.listen("tcp://")
 
         async with rpc(server.address, serializers=["msgpack"]) as r:
-            with pytest.raises(TypeError):
-                await r.echo(x=to_serialize(inc))
+            await r.echo(x=to_serialize(inc))
```

Comment:
These sorts of changes are probably not ok.  They fundamentally change the intent of the test, which is to ensure that things like this can be disallowed.

Reply:
This has been fixed